src/elm/Types/DeviceEvent.elm

```diff
@@ -139,6 +143,19 @@ valueParamsDecoder =
         (Decode.field "value" AstarteValue.decoder)
 
 
+unsetPropertyDecoder : Decoder PathParams
+unsetPropertyDecoder =
+    Decode.field "value" (Decode.nullable Decode.value)
+    |> Decode.andThen (\val ->
+        case val of
+            Nothing ->
+                pathParamsDecoder
+
+            _ ->
+                Decode.fail "invalid incomming_data"
```

Comment:
Drop an `m` from `incomming`

Reply:
And, thinking about it, I suppose it should be "invalid unset property" or something like that, if you want to tell it apart

Reply:
can you really tell if it's an invalid unset? the only known thing is that type is `incoming_data` and its value is not null